app/sidekiq/evss/disability_compensation_form/submit_form526.rb

```diff
@@ -107,8 +107,7 @@ def perform(submission_id) # rubocop:disable Metrics/MethodLength
           begin
             submit_to_claims_api = submission.claims_api?
             # send submission data to either EVSS or Lighthouse (LH)
-            response = if Flipper.enabled?(:disability_compensation_lighthouse_submit_migration, user) ||
-                          submit_to_claims_api # right operand evaluation not needed once fully migrated to LH
+            response = if submit_to_claims_api # not needed once fully migrated to LH
```

Comment:
In addition to removing the `user` variable, there's no need to assign the `submit_to_claims_api` variable as the only other place it was used was as an argument in the now-defunct `generate_options_hash` method.

Reply:
ah thank you, I missed that! PR updated